### PR TITLE
Camera to director shot deletion

### PR DESCRIPTION
**Pull request by geenen124**
Also repaired loading of DirectorShots and CameraShots. These were circumventing the controller's methods and duplicating code with unwanted side effects.

### CODE REVIEW

File: src/main/java/control/ProjectController.java [hunk 1]
```diff
@@ -297,12 +297,12 @@ public void load(File file) {
                             controllerManager.getScriptingProject()
                                     .getCameraTimelines()
                                     .size(), false));
+//            controllerManager.getTimelineControl()
+//                    .setNumTimelines(controllerManager.getScriptingProject()
```
Comment (mennoo1996) [inline src/main/java/control/ProjectController.java]: Please delete this outcommented code if it is not necessary anymore

Comment (geenen124) [inline src/main/java/control/ProjectController.java]: Whoops

**Comment by bart-de-jonge** [comment]
Looks good

**Comment by mennoo1996** [comment]
Great!

**Pull request merged**